action.yml

```diff
@@ -2,52 +2,51 @@ name: Update Deployment Board
+    description: The repo the deployment will be created in, i.e. [org|owner]/[repo name]
     required: true
   ref:
-    description: 'The branch, tag or SHA that was deployed'
+    description: The branch, tag or SHA that was deployed
     required: true
-  ref-type:
-    description: 'The type of ref that was deployed  [branch|tag|sha].  If not provided the action will use some regex patterns to identify the type.'
-    required: false
-  deployable-type:
-    description: 'The type of artifact that was deployed like MFE, BFF, API, SVC, DB, etc.  Useful when a release with one tag contains multiple deployable artifacts that are deployed separately.'
-    required: false
-  deploy-status:
-    description: 'The status of the deployment [success|failure|cancelled|skipped]'
+  deployment-status:
+    description: The status of the deployment [error|failure|success]
     required: true
-  deploy-label:
-    description: 'The optional label of the deployment [deleted|destroyed|<your-custom-label>]'
+  deployment-message:
+    description: Any message to be delivered along side of the status
     required: false
-  enable-deployment-slot-tracking:
-    description: 'Enable App Service deployment slot tracking on deployment board? [true|false]'
-    required: false
-    default: 'false'
-  slot-swapped-with-production-slot:
-    description: 'Did this deployment swap slots with production slot? Set to true for swapping slot immediately with production slot. [true|false]'
+  deployment-auto-inactivate:
+    description: |
+      Automatically mark prior environment deployments inactive, only use 'true' for environments that do not
+      have multiple instances, expects 'true' or 'false', defaults to 'false'
     required: false
     default: 'false'
-  target-slot:
-    description: 'The target slot that was deployed to [production|blue|yellow|canary|red|loadtest|predeploy|your-custom-slot]'
-    required: false
-    default: 'production'
-  source-slot:
-    description: 'The source slot that was deployed to [production|blue|yellow|canary|red|loadtest|predeploy|your-custom-slot]'
-    required: false
-    default: 'production'
-  timezone:
-    description: 'IANA time zone name (e.g. America/Denver) to display dates in.  If timezone is not provided, dates will be shown in UTC'
-    required: false
+  entities:
+    description: The entities that are affected by the deployment, i.e. ["proj-app", "proj-infrastruction", "proj-db"]
+    required: true
+  instance:
+    description: The target deployment server or app service and availability zone, i.e. "NA26", "NA26-slot1", "NA27-blue"
+    required: true
+  workflow-run-url:
+    description: The url of the workflow run, i.e."https://github.com/[owner]/[repo]/actions/runs/[workflow run id]"
+    required: true
+  workflow-task:
+    description: The task designation for the deployment, defaults to `workflowdeploy`                                     |
+    required: true
+    default: workflowdeploy
```

Comment:
I don't think this should be an input variable. If we decide in the future we need to record some other `task` in deployment objects, we could add the input at that time, but more than likely, it would also be a reason to create a different custom action for it anyway.